.swiftlint.yml

```diff
@@ -34,12 +34,10 @@ opt_in_rules:
   - empty_string
   - fatal_error_message
   - first_where
-  - modifier_order
   - toggle_bool
   - identical_operands
   - joined_default_parameter
   - unavailable_function
-  - overridden_super_call
```

Comment:
Why are those options removed?

Reply:
They were annoying and incorrect. 

**Order of the override:** The order of the override is not that important, and it was the opposite of Xcode's autocomplete. 

**Overriden super call:** This was incorrect. This meant that we had to call `super` every time. In some cases we might to want to completely override the super function, just like we did here with the Search view.